docs/mainnet.md

```diff
@@ -4,7 +4,7 @@ The swap operates on a maker/taker paradigm, where one party is a "market maker"
 
 Currently, due to protocol limitations, XMR holders who want ETH can only act as makers, and ETH holders who want XMR can only act as takers.
 
-> **Note:** a swap on mainnet currently takes around 20-25 minutes due to block time.
+> **Note:** a swap on mainnet currently take around 20-25 minutes due to block time.
```

Comment:
change this back?

Reply:
The noun in this sentence, `swap`, is singular. We could make it plural. I was just fixing the grammar with minimal changes.

Reply:
Will fix!